src/style.css

```
@@ -0,0 +1,25 @@
+.container {
+  /* margin: 32px 0 0 20px; */
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin: 0 2px;
```

To fix tests you need to style stas like this:
```suggestion
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
  width: 16px;
  height: 16px;
  margin-right: 4px;
```